### PR TITLE
update  script and fix the mistakes I have done

### PR DESCRIPTION
**Pull request by cedlemo**

### CODE REVIEW

File: gtk3/sample/misc/aboutdialog2.rb [hunk 1]
```diff
@@ -2,7 +2,7 @@
 =begin
   aboutdialog2.rb - Ruby/GTK sample script.
 
-  Copyright (c) 2005-2015 Ruby-GNOME2 Project Team
+  Copyright (c) 2006-2015 Ruby-GNOME2 Project Team
```
Comment (kou) [inline gtk3/sample/misc/aboutdialog2.rb]: Please don't change copyright start year.

Comment (cedlemo) [inline gtk3/sample/misc/aboutdialog2.rb]: The original started at 2006 . The last commit is a fix to my previous error on this copyright.

Comment (kou) [inline gtk3/sample/misc/aboutdialog2.rb]: Really?
https://github.com/cedlemo/ruby-gnome2/commit/5f4ebc07a4eed20529347698b373244999bb1127#diff-143503283065af58477f247b38700298L5 says

```text
 Copyright (c) 2005,2006 Ruby-GNOME2 Project Team 
```

**Comment by cedlemo** [comment]
Is there something wrong with my pull requests? Should I continue to update the sample scripts or would you want that I make any other modifications?

**Comment by kou** [comment]
> Is there something wrong with my pull requests?

No. Just I was busy.

> Should I continue to update the sample scripts or would you want that I make any other modifications?

Please continue!

**Pull request merged**